sites/optimism/package.json

```
@@ -17,24 +17,25 @@
     "build": "pnpm build:assets && docusaurus build --out-dir build"
   },
   "dependencies": {
-    "@handbook/common-config": "workspace:*",
+    "@cookbookdev/docsbot": "^4.25.15",
```

let's lock the version
```suggestion
    "@cookbookdev/docsbot": "4.25.15",
```